oracle/assets/configuration/spec.yaml

```
@@ -21,6 +21,17 @@ files:
       required: true
       value:
         type: string
+    - name: protocol
+      description: |
+        The protocol to connect to the Oracle Database Server. Valid protocols include TCP and TCPS.
+
+        When connecting to Oracle Database via JDBC, `jdbc_truststore` and `jdbc_truststore_type` are required.
```

is this only for TCPS?

```suggestion
        If using the TCPS protocol and connecting to Oracle Database via JDBC, `jdbc_truststore` and `jdbc_truststore_type` are required.
```